material_studies/rally/lib/tabs/accounts.dart

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019-present the Flutter authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+import 'package:rally/data.dart';
+import 'package:rally/finance.dart';
+import 'package:rally/charts/pie_chart.dart';
+
+/// A page that shows a summary of accounts.
+class AccountsView extends StatelessWidget {
+  final List<AccountData> items = DummyDataService.getAccountDataList();
+
+  @override
+  Widget build(BuildContext context) {
+    double balanceTotal = sumAccountDataPrimaryAmount(items);
+    List<RallyPieChartSegment> segments = fromAccountItems(items);
```

Comment:
This local doesn't need to exist. It can just be inlined on line 33.

Reply:
Done